### ISSUE TITLE
screenshot-jpeg-quality settings is ineffective

### ISSUE THREAD

**Issue by etienned** [mpv-player/mpv#11850]
Information:
- mpv version : 0.35.1
- Platform and Version : macos 12.6.6
- Source of the mpv binary : homebrew cask

### Reproduction steps

1. Change only the screenshot jpeg quality with `--screenshot-jpeg-quality=INT` on the command line or `screenshot-jpeg-quality=INT` in the mpv.conf, putting different values (between 0 and 100). No other settings are set.
2. Open a video, choose a specific frame and make a screenshot by pressing "s".
3. Compare the size of the resulting JPEGs (of the same frame) for different values, it will always be the same.  

### Expected behavior

File size (and quality) of the JPEG screenshots should change in function of the quality settings.

### Actual behavior

File size (and quality) of the JPEG screenshots are always the same, whatever the JPEG quality is.

### Log file

[output.txt](https://github.com/mpv-player/mpv/files/11900334/output.txt)

### Note

The jpeg quality setting was working in older versions, but unfortunately I didn't notice when the change happened. For information, the PNG compression settings is working.

**Comment by llyyr** [comment]
can't reproduce, this works for me on linux

```
109 KB 11 seconds ago mpv-shot0001.jpg
426 KB now            mpv-shot0002.jpg
```

**Comment by llyyr** [comment]
I can reproduce this by building mpv with -Djpeg=disabled, which means your cask isn't built with libjpeg. Contract whoever builds those (I guess [here?])(https://laboratory.stolendata.net/~djinn/mpv_osx/) or build mpv yourself

**Comment by sfan5** [comment]
Note that this can still be considered a bug, whether mpv uses libjpeg or ffmpeg to write screenshots should be transparent to the end user.

**Comment by sfan5** [comment]
fixed by 76589a5b34ba9769372cd8f233890ab892fd27ad

**Comment by etienned** [comment]
Wow, that was really fast, I'm sincerely impress! Thanks for your work.
